test/lib/print-test.js

```diff
@@ -90,11 +90,9 @@ test('print: prints verbs in cyan', t => {
 })
 
 test('print: prints texts in cyan', t => {
```

Comment:
Good catch. Yellow was finally intended (as Date, since this is no longer a function). Fixing the test description.